### DISCUSSION TITLE
API Documentation of /user/v3/search

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Lern/Community#75] [Documentation] [answered]
Hello All,

I am referring tio documentation of `/user/v3/search` API [here](https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service/api-documentation/user-management)

On doing some trial I found out that ths API has an `offset` parameter which can be used but, the documentation does not mention more details about it . Can it be added ?

Apart from that the document example shows filters set for search on emailId but this API returns masked emails how does it perform search when email is passed? Please help me understand it.

**Comment by rhwarrier** [comment]
@Hari-stackroute  can you pls take a look. cc: @reshmi-nair

**Reply by reshmi-nair** [reply, marked as answer]
@Xitija We will update the details on offset in the documentation.

While giving email in the request for search, first it is encrypted and then ES search is performed with encrypted data, and while returning search results, this encrypted fields are removed from response.
